mlflow/pyfunc/__init__.py

```diff
@@ -977,6 +977,7 @@ def spark_udf(spark, model_uri, result_type="double", env_manager="local"):
                            that was used to train the model. Note that environment is only restored
                            in the context of the PySpark UDF; the software environment outside of
                            the UDF is unaffected.
```

Comment:
This note about the environment only being restored for UDF also applies to virtualenv. Instead of repeating it for `conda` and `virtualenv`, can we move it above the list?

```
    :param env_manager: The environment manager to use in order to create the
                        software environment for model inference. Note that environment is only restored
                        in the context of the PySpark UDF; the software environment outside of the UDF is 
                         unaffected. Default value is ``local``, and the following values are supported:
```